src/main/java/org/folio/circulation/resources/DeclareLostResource.java

```diff
@@ -42,9 +42,9 @@ private void declareLost(RoutingContext routingContext) {
       loanRepository.getById(request.getLoanId())
         .thenApply(LoanValidator::refuseWhenLoanIsClosed)
         .thenApply(loan -> declareItemLost(loan, request))
+        .thenCompose(r -> r.after(storeLoanAndItem::updateLoanAndItemInStorage))
         .thenCompose(r -> r.after(loan -> lostItemFeeService
           .chargeLostItemFees(loan, request, context.getUserId()))))
-      .thenCompose(r -> r.after(storeLoanAndItem::updateLoanAndItemInStorage))
```

Comment:
Has this moved because updating the loan and item is considered more important than charging the fees?

Reply:
It has been moved to persist the declared lost action. Because the next method can close the loan.

Reply:
Does that mean that the `chargeLostItemFees` triggers one of the loan updates and `updateLoanAndItemInStorage` triggers the other?

Reply:
yes, correct.